shared/transforms/shared_table-srgmap.xslt

```diff
@@ -31,16 +31,24 @@
 
 		<table>
 			<thead>
-				<td>SRG ID</td>
-				<td>CCI ID</td>
-				<td>SRG Title</td>
-				<td>SRG Description</td>
+				<td>CCI</td>
+				<td>SRGID</td>
+				<td>STIGID</td>
+				<td>SRG Requirement</td>
 				<xsl:choose>
 					<xsl:when test="$flat">
-						<td>Rule ID</td>
-						<td>Rule Title</td>
-						<td>Rule Desc</td>
-						<td>Rule Check</td>
+						<td>Requirement</td> <!-- XCCDF RULE ID -->
```

Comment:
@shawndwells do we only want this for flat tables?

Reply:
For now. Can incrementally adjust other outputs.